lib/services/local/externalcloudaudit.go

```diff
@@ -20,15 +20,23 @@ import (
 	"context"
 	"time"
 
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/service/athena"
+	"github.com/aws/aws-sdk-go-v2/service/glue"
+	"github.com/aws/aws-sdk-go-v2/service/s3"
 	"github.com/gravitational/trace"
 	"github.com/sirupsen/logrus"
 
 	"github.com/gravitational/teleport/api/types/externalcloudaudit"
 	"github.com/gravitational/teleport/api/types/header"
 	"github.com/gravitational/teleport/lib/backend"
+	ecaint "github.com/gravitational/teleport/lib/integrations/externalcloudaudit"
```

Comment:
I'm going to suggest that `lib/services/local` isn't the right place for this logic, the new imports are what first prompted me. `lib/serivces/local` is mostly a thin wrapper over the backend and doesn't really import SDK stuff or too many other libraries.

We do need RPCs for it though, and I see how `local.ExternalCloudAuditService` and `api/client/externalcloudaudit.Client` both having to implement `services.ExternalCloudAudits` would make you write it this way - but we don't have to use the interface.

I suggest removing the new methods from the interface, and calling all this code from the gRPC service layer instead of the local storage layer. Ping me if you want to chat about it.